internal/status/cache.go

```diff
@@ -18,12 +18,22 @@ package status
 import (
 	"time"
 
-	projcontour "github.com/projectcontour/contour/apis/projectcontour/v1"
+	contour_api_v1 "github.com/projectcontour/contour/apis/projectcontour/v1"
 	"github.com/projectcontour/contour/internal/k8s"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+type ProxyStatus string
+
+const (
+	ProxyStatusValid    ProxyStatus = "valid"
+	ProxyStatusInvalid  ProxyStatus = "invalid"
+	ProxyStatusOrphaned ProxyStatus = "orphaned"
```

Comment:
these constants are also defined in internal/k8s/status.go, can we consolidate to one set?

Reply:
Done.